### PR TITLE
projects link

### PR DESCRIPTION
**Pull request by 21Danson**
## Summary of Changes

first turn in of projects website link

## Imagery

used temporary pictures to depict what future projects will look like on slider

### CODE REVIEW

**Review by GCUCoding — COMMENTED** [review]
Fixed a couple of issues that GitHub created, will look into it more later. Great job so far!

File: projects.html [hunk 1]
```diff
@@ -1,25 +1,131 @@
...
+<br>
+
+<div style="text-align:center">
+  <span class="dot"></span> 
+  <span class="dot"></span> 
+  <span class="dot"></span> 
+</div>
+
+<script>
+var slideIndex = 0;
```
Comment (elijaholmos) [inline projects.html:85]:
```suggestion
let slideIndex = 0;
```

File: projects.html [hunk 2]
```diff
@@ -1,25 +1,131 @@
...
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("dot");
```
Comment (elijaholmos) [inline projects.html:91]:
```suggestion
  let i;
  let slides = document.getElementsByClassName("mySlides");
  let dots = document.getElementsByClassName("dot");
```

File: projects.html [hunk 3]
```diff
@@ -1,25 +1,131 @@
...
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none";  
+  }
```
Comment (elijaholmos) [inline projects.html:94]:
```suggestion
  for (const slide of slides) {
    slide.style.display = "none";  
  }
```

File: projects.html [hunk 4]
```diff
@@ -1,25 +1,131 @@
...
+  slideIndex++;
+  if (slideIndex > slides.length) {slideIndex = 1}
```
Comment (elijaholmos) [inline projects.html:96]:
```suggestion
  if (slideIndex > slides.length) slideIndex = 1;
```

File: projects.html [hunk 5]
```diff
@@ -1,25 +1,131 @@
...
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none";  
+  }
+  slideIndex++;
+  if (slideIndex > slides.length) {slideIndex = 1}    
+  for (i = 0; i < dots.length; i++) {
+    dots[i].className = dots[i].className.replace(" active", "");
+  }
+  slides[slideIndex-1].style.display = "block";  
+  dots[slideIndex-1].className += " active";
```
Comment (elijaholmos) [inline projects.html:101]: We can actually use DOM `classList` property here:
```suggestion
  for (const dot of dots) {
    dot.classList.remove('active');
  }
  slides[slideIndex-1].style.display = "block";  
  dots[slideIndex-1].classList.add('active');
  ```

**Review by elijaholmos — CHANGES_REQUESTED** [review]
Added some JS changes

**Review by elijaholmos — APPROVED** [review]

**Pull request merged**